### PR TITLE
Planets spawning inside black hole / each other

### LINKED ISSUE #91: Planet's Spawning inside Blackhole
Noticed this while working on updated planet indication, Sometimes planets can spawn directly inside the blackhole and are destroyed immediately when the game is started. There should be some kind of check to prevent this from happening. Perhaps similar to how planet on planet collisions are prevented.

### PR DESCRIPTION
**Pull request by PBWiseman**
This branch is to fix the bugs where planets either spawn too close to the black hole and are instantly killed or spawn inside other planets. This will fix #91 and #84

Edit: I also made improvements to how the planets orbit and rotate while I was working in the scripts

### CODE REVIEW

**Comment by PBWiseman** [comment]
Ready for review. Spawning in black hole and spawning collided bugs should be fixed. Needed an overhaul of the spawning practice. Now it spawns each planet at a set X position and then rotates them around the Z and Y axis. While I was doing this I also improved the way the planets orbit. Planets now orbit at greater speeds when closer to the center of the solar system and can orbit at an angle instead of just in a circle.

**Review by ChaseBENNC9 — CHANGES_REQUESTED** [review]
The planets no longer spawn inside each other. However I noticed that they can move extremely close to each other 
image
in some cases this causes them to collide again
image

I like your code comments and the new comment headers will help with identifying the purpose of the scripts

File: Assets/Scripts/Planets/PlanetSpawn.cs [hunk 1]
```diff
@@ -1,91 +1,55 @@
...
-
-            }while (!isNotCollision); //if its not true then it will try again
-            //TODO add loop counter to while loop
-            loopCounter = 0;
-            planetTemp = Instantiate(planetPrefab, new Vector3(spawnX, spawnY, spawnZ), Quaternion.identity);
-            planetTemp.transform.localScale = new Vector3(randScale, randScale, randScale);
-            planetTemp.transform.parent = planetParent.transform;
+            xDistance = i * ORBITAREA + BUFFER; //Set xDistance
+            scale = Random.Range(SCALEMIN, SCALEMAX); //Random scale
+            planetTemp = Instantiate(
```
Comment (erokasyuart) [inline Assets/Scripts/Planets/PlanetSpawn.cs:41]: Could you either make this one line or make all the other lines (like random range on line 49) expanded like you've done here. For consistency sake :)

**Review by erokasyuart — APPROVED** [review]
I don't have an issue with this new approach to planet spawning. If hardcoding their position is a solution to stop them from spawning inside of each other and in the black hole then it's a good solution :)

File: Assets/Scripts/Planets/PlanetSpawn.cs (hunk 1 above)
Comment (PBWiseman) [reply Assets/Scripts/Planets/PlanetSpawn.cs:41]: This is done automatically through [CSharpier](https://csharpier.com/), the program we are using for code formatting, once a list of arguments reaches a certain length.

**Comment by PBWiseman** [comment]
Reduced from 10 planets to 8, slightly shrunk the sizes, and slightly increased the gap between them. This should ensure planets do not collide anymore. I also removed the massive gravity as having constant raycasts to all planets was an unneeded performance loss for an almost unnoticeable gravity amount on the player.

**Review by ChaseBENNC9 — APPROVED** [review]
All Concerns have been addressed.

**Pull request merged**